corehq/apps/reminders/templates/reminders/partial/scheduled_reminders.html

```diff
@@ -83,13 +165,7 @@
     </style>
 {% endblock %}
 
-{% block subsection-title %}
-<li class="active">
-    <a href="#">Reminder Calendar</a>
-</li>
-{% endblock %}
-
-{% block main_column %}
+{% block page_content %}
     {% include "style/bootstrap2/partials/time_notice.html" %}
```

Comment:
I would leave this alone for right now. I can handle moving the file around in a separate PR. This issue will likely come up in the other reminders pages, so I'll wait until that work is done to reduce merge conflictss